distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestSpatial.java

```diff
@@ -620,6 +672,57 @@ public void testWfs11Boolean() {
     assertMetacards(responseXml, 2, WFS_11_SOURCE_ID, "roads");
   }
 
+  /**
+   * A downstream project encountered a situation where a WFS server would return an exception
+   * report for the hit count, but still return valid results. The SourceResponse contract allows
+   * the hit count to be -1 if it is unknown. {@link #setupWfs11QueryHitsExceptionHitCount} {@link
+   * #setupWfs11QueryHitsExceptionResults}
+   */
+  @Test
+  public void testWfsHitsException() {
+
+    String response =
+        sendCswQuery(
```

Comment:
❓ CSW?

Reply:
I needed to use an endpoint that would send back the total hits, which is -1 in this case.

Reply:
gotcha - thanks